builtin/settingtypes.txt

```diff
@@ -871,10 +893,17 @@ enable_console (Enable console window) bool false
 
 [Sound]
 
+#    Enables the sound system.
+#    If disabled, this completely disables all sounds everywhere and the in-game sound controls will be non-functional.
+#    Changing this setting requires a restart.
 enable_sound (Sound) bool true
 
+#    Volume of all sounds.
+#    Requires the sound system to be enabled.
 sound_volume (Volume) float 0.7 0.0 1.0
 
+#    Whether to mute sounds. You can unmute sounds at any time, unless the sound system is disabled (enable_sound=false).
```

Comment:
This and L897 are too long. Please break around 80 characters.

Reply:
Agreed. Although comments now auto-wrap in the formspec, the convention used in settingtypes.txt is to format comment lines fairly short so that they do not auto-wrap in the formspec (with the default game window size 1024x600). Shorter lines have benefits for development.
Maybe we'll change our minds in future but for now i prefer consistency and file readability.